basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/jobs/SiteMembershipsSyncJob.java

```diff
@@ -31,18 +34,30 @@ public void setSiteMembershipsSynchroniser(SiteMembershipsSynchroniser siteMembe
 	public void execute(JobExecutionContext context) throws JobExecutionException {
 
         M_log.info("SiteMembershipsSyncJob.execute");
+        List<String> emailTrustedConsumersList =null;
```

Comment:
This duplicates code in ProviderServlet - should have only one place to determine if isEmailTrustedConsumer. I'd suggest overloading ProviderServlet.isEmailTrustedConsumer( String consumerKey )